### ISSUE TITLE
SyntaxError: Unexpected token } after install

### ISSUE THREAD

**Issue by dvabuzyarov** [nonara/ts-patch#2]
"typescript": "3.2.4"

/node_modules/typescript/lib/tsc.js:120404
});
^

SyntaxError: Unexpected token }
    at new Script (vm.js:79:7)
    at createScript (vm.js:251:10)
    at Object.runInThisContext (vm.js:303:10)
    at Module._compile (internal/modules/cjs/loader.js:656:28)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)

**Comment by nonara** [comment]
Seems to be working here. Here's what I did.

**Setup**:
```
mkdir newDir
cd newDir
npm i typescript@3.2.4
npm i ts-patch -g
ts-patch i
touch m.ts
```

Output:
```
```

Can you provide me with some steps to reproduce?

**Comment by dvabuzyarov** [comment]
Thanks you very much. I will test it today latter on and proved the result.

**Comment by dvabuzyarov** [comment]
Checking TypeScript v3.2.4 installation in moq.ts/node_modules/typescript

[+] tsc.js is patched with ts-patch version 1.0.4.
[+] tsserverlibrary.js is patched with ts-patch version 1.0.4.
[+] typescript.js is patched with ts-patch version 1.0.4.
[+] typescriptServices.js is patched with ts-patch version 1.0.4.

tsc -v
Version 3.6.3

The error is still persisted. 

When the globally installed typescript was updated to 3.7.2 than the error disappeared. So obviously the combination of the globally installed tsc and local version leads to the error.
As for me it is fine to update the global tsc.

Thanks for you work.

**Comment by nonara** [comment]
Whoops. Looks like I missed the fact that npm was pointing tsc command to the global install!

Just to be safe I tried it with 3.2.4 and 3.6.3 using `node node_modules\typescript\bin\tsc m.ts` Both worked. 

Maybe your global tsc may was corrupted before the update. Anyway, glad it's working for you!
